hack/make/.integration-daemon-start

```diff
@@ -50,6 +50,11 @@ if [ "$DOCKER_REMAP_ROOT" ]; then
 	extra_params="--userns-remap $DOCKER_REMAP_ROOT"
 fi
 
+if [  "$DOCKER_EXPERIMENTAL" ]; then
+	echo >&2 '# DOCKER_EXPERIMENTAL is set: starting daemon with experimental features enabled! '
+	extra_params="$extra_params --experimental"
+fi
```

Comment:
Without it the experimental endpoint are not "exposed" to the clients.

However, if there''s code elsewhere that is supposed to be experimental and isn't guard by the proper check, testing both with and without the flag should help catch it.